main.go

```diff
@@ -1,4 +1,4 @@
-package main // import "go.uber.org/sally"
```

Comment:
Please add this canonical import back - the correct path for this project is `go.uber.org/sally`, this enforces that.

https://texlution.com/post/golang-canonical-import-paths/